packages/components/messages/src/index.ts

```diff
@@ -1,5 +1,6 @@
 export { default as ErrorMessage } from './error-message';
 export { default as WarningMessage } from './warning-message';
+export { default as AdditionalInfoMessage } from './additional-info-message';
```

Comment:
types should also be exported from here, otherwise they won't be available for direct import from `"@commercetools-uikit/messages"`

Reply:
I updated it now